sys/purge.sh

```diff
@@ -6,30 +6,25 @@ if [ -z "${MAKE}" ]; then
 	[ $? = 0 ] && MAKE=gmake
 fi
 
-if [ "$1" = "distro" ]; then
-	# TODO: Query the user before taking any action
-	if [ -x /usr/bin/apt-get ] ; then
-		sudo apt-get remove radare2
-		sudo apt-get remove libradare2-common
-		sudo apt-get remove --auto-remove libradare2-common
-		sudo apt-get purge libradare2-common
-		sudo apt-get purge --auto-remove libradare2-common
-		sudo apt-get remove libradare2
-		sudo apt-get remove --auto-remove libradare2
-		sudo apt-get purge libradare2
-		sudo apt-get purge --auto-remove libradare2
-	fi
-	# TODO: support brew
-	# TODO: support archlinux
-	# TODO: support gentoo
-	exit 0
+echo -n "All files related to current and previous installations 
```

Comment:
echo -n is not portable. use printf instead, but without -n is also fine  for me